src/utils/core.ts

```diff
@@ -1730,25 +1730,20 @@ export const COMMANDS: {
   },
   {
     text: "REPEAT",
-    delayArgs: true,
-    help: "Repeats the current block a number of specified times\n\n1. Number of times for repeat",
-    handler: (repeatArg = "1", content = "") =>
-      proccessBlockText(repeatArg)
-        .then(([{ text }]) =>
-          Array(Number(text) || 1)
-            .fill(content)
-            .map((s) => () => proccessBlockText(s))
-            .reduce(
-              (prev, cur) =>
-                prev.then((p) =>
-                  cur().then((c) => {
-                    return [...p, c];
-                  })
-                ),
-              Promise.resolve([] as InputTextNode[][])
-            )
-        )
-        .then((s) => s.flatMap((c) => c)),
+    delayArgs: [true, true, false],
+    help: "Repeats the current block a specified number of times\n\n1. Number of times for repeat",
+    handler: async (repeatArg = "1", content = "", ...values) => {
```

Comment:
settings `repeatArg` to `false` was overriding `content`

eg: `<%REPEAT:<%CHILDREN:((uid)),<%SMARTBLOCK:runMe%>%>` resulted in `content` = `child2`

Reply:
This is still the case, even changing `.map` to `.reduce` @dvargas92495 

https://www.loom.com/share/07aa41eb1b48496586b7c9a4e61f6f5d